<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContextMenu</class>
 <widget class="QWidget" name="ContextMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>398</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>210</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="title" stdset="0">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="zoomLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="ActionButton" name="fitWindow">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="accessibleName">
        <string>IconButton</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/res/icons/buttons/view-fullscreen16.png</normaloff>:/res/icons/buttons/view-fullscreen16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ActionButton" name="fitWidth">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="accessibleName">
        <string>IconButton</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/res/icons/buttons/fit-width16.png</normaloff>:/res/icons/buttons/fit-width16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ActionButton" name="zoomOriginal">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="accessibleName">
        <string>IconButton</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/res/icons/buttons/zoom-original16.png</normaloff>:/res/icons/buttons/zoom-original16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ActionButton" name="zoomIn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="accessibleName">
        <string>IconButton</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/res/icons/buttons/zoom-in16.png</normaloff>:/res/icons/buttons/zoom-in16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ActionButton" name="zoomOut">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="accessibleName">
        <string>IconButton</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/res/icons/buttons/zoom-out16.png</normaloff>:/res/icons/buttons/zoom-out16.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="actionsLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="ContextMenuItem" name="rotateLeft" native="true"/>
     </item>
     <item>
      <widget class="ContextMenuItem" name="rotateRight" native="true"/>
     </item>
     <item>
      <widget class="ContextMenuItem" name="flipH" native="true"/>
     </item>
     <item>
      <widget class="ContextMenuItem" name="flipV" native="true"/>
     </item>
     <item>
      <widget class="ContextMenuItem" name="crop" native="true"/>
     </item>
     <item>
      <widget class="ContextMenuItem" name="resize" native="true"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>6</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="ContextMenuItem" name="copy" native="true"/>
     </item>
     <item>
      <widget class="ContextMenuItem" name="move" native="true"/>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>6</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="ContextMenuItem" name="trash" native="true"/>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>6</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="ContextMenuItem" name="open" native="true"/>
     </item>
     <item>
      <widget class="ContextMenuItem" name="folderView" native="true"/>
     </item>
     <item>
      <widget class="ContextMenuItem" name="settings" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ActionButton</class>
   <extends>QPushButton</extends>
   <header>gui/customwidgets/actionbutton.h</header>
  </customwidget>
  <customwidget>
   <class>ContextMenuItem</class>
   <extends>QWidget</extends>
   <header>gui/customwidgets/contextmenuitem.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
